public/pages/Findings/components/FindingsTable/ThreatIntelFindingsTable.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiBasicTableColumn, EuiButtonIcon, EuiInMemoryTable, EuiToolTip } from '@elastic/eui';
+import { ThreatIntelFinding } from '../../../../../types';
+import React from 'react';
+import { renderTime } from '../../../../utils/helpers';
+import { DEFAULT_EMPTY_DATA } from '../../../../utils/constants';
+import { DataStore } from '../../../../store/DataStore';
+
+export interface ThreatIntelFindingsTableProps {
+  findingItems: ThreatIntelFinding[];
+}
+
+export const ThreatIntelFindingsTable: React.FC<ThreatIntelFindingsTableProps> = ({
+  findingItems,
+}) => {
+  const columns: EuiBasicTableColumn<ThreatIntelFinding>[] = [
+    {
+      name: 'Time',
+      field: 'timestamp',
+      render: (timestamp: number) => renderTime(timestamp),
+    },
+    {
+      name: 'Indicator of compromise',
+      field: 'ioc_value',
+    },
+    {
+      name: 'Indicator type',
+      field: 'ioc_type',
+    },
+    {
+      name: 'Threat intel source',
+      field: 'ioc_feed_ids',
+      render: (ioc_feed_ids: ThreatIntelFinding['ioc_feed_ids']) => {
```

Comment:
what if ioc_feed_ids come as null or undefined?

Reply:
we can do something like this?

```
render: (ioc_feed_ids: ThreatIntelFinding['ioc_feed_ids']) => {
    if (!ioc_feed_ids || ioc_feed_ids.length === 0) {
      return <span>{DEFAULT_EMPTY_DATA}</span>;
    }

    return <span>{ioc_feed_ids[0]?.feed_id ?? DEFAULT_EMPTY_DATA}</span>;
  },
```

Reply:
That would be a contract violation by the backend API, so I don't think we should add the validation to do extra work